packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/overview/edit-name/controller.js

```diff
@@ -0,0 +1,54 @@
+export default class OctaviaLoadBalancerEditNameCtrl {
+  /* @ngInject */
+  constructor(OctaviaLoadBalancerPoolsService, Alerter, $translate) {
+    this.isLoading = false;
+    this.OctaviaLoadBalancerPoolsService = OctaviaLoadBalancerPoolsService;
+    this.Alerter = Alerter;
+    this.$translate = $translate;
+  }
+
+  $onInit() {
+    this.name = this.pool.name;
+  }
+
+  cancel() {
+    this.goBack();
```

Comment:
You are missing the tracking here

Reply:
Vu avec @JacquesLarique pas de tracking spécifié dans le fichier excel pour l'instant donc on enlève.